docs/flatcar-linux/kvm-libvirt.md

```diff
@@ -0,0 +1,349 @@
+# KVM libvirt
+
+In this tutorial, we'll create a Kubernetes cluster with Flatcar Linux inside VMs on our local machine.
+
+We'll declare the cluster using the Lokomotive Terraform module for the KVM libvirt platform.
+
+Controllers are provisioned to run an `etcd-member` peer and a `kubelet` service.
+Workers run just a `kubelet` service. A one-time [bootkube](https://github.com/kubernetes-incubator/bootkube)
+bootstrap schedules the `apiserver`, `scheduler`, `controller-manager`, and `coredns` on controllers
+and schedules `kube-proxy` and `calico` (or `flannel`) on every node.
+A generated `kubeconfig` provides `kubectl` access to the cluster.
+
+## Requirements
+
+* `qemu-img`, `wget`, and `bunzip2` binaries installed
+* At least 4 GB of free RAM
+* Running libvirtd system service, see [libvirtd Setup](#libvirtd-setup)
+* A user in the `libvirt` group (to create virtual machines with libvirt and access `/dev/kvm`), see [User Setup]
+* Terraform v0.11.x, [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct),
+  and [terraform-provider-libvirt](https://github.com/dmacvicar/terraform-provider-libvirt)
+  installed locally
+
+## Prepare VM Image
+
+Download the Flatcar image. The example below uses the "Edge" Flatcar channel.
+You need to increase its size depending on your use case because the initial size
+is only enough for some small pods to be loaded. Starting pods with normal-sized
+container images may already fail because the disk space will be filled.
+That's why the example below directly increases the image size by 5 GB to make
+sure that a couple of container images fit on the node.
+
+```sh
+$ sudo dnf install wget bzip2 qemu-img  # or sudo apt install wget bzip2 qemu-utils
+$ wget https://edge.release.flatcar-linux.net/amd64-usr/2135.99.0/flatcar_production_qemu_image.img.bz2
```

Comment:
Maybe we can bundle all image-related step as `command1 && command2 && command3` so user just need to copy and execute one line?

I would also change the version to `current`, so user don't have to wonder which version is the latest.

Reply:
I'll prefer we leave the commands as they are though.

Reply:
Thanks!